brax/physics/forces.py

```diff
@@ -21,47 +21,56 @@
 from brax import pytree
 from brax.physics import bodies
 from brax.physics import config_pb2
-from brax.physics.base import P, QP
-
+from brax.physics.base import P, QP, vec_to_arr
 
 @pytree.register
 class Thruster:
-  """Applies a force to a body."""
+  """Applies an external force or torque to a body."""
 
   def __init__(self, forces: List[config_pb2.Force], body: bodies.Body,
-               act_index: List[Tuple[int, int]]):
+               force_index: List[Tuple[int, int]],
+               torque_index: List[Tuple[int, int]]):
     """Creates an actuator that applies torque to a joint given an act array.
 
     Args:
       forces: list of forces (all of the same type) to batch together
       body: (batched) bodfies for this force to act upon
-      act_index: indices from the act array that drive this force
+      force_index: indices from the act array that drive this force
+      toque_index: indices from the act array that drive this torque
     """
     self.body = jp.take(body, [body.index[f.body] for f in forces])
     self.strength = jp.array([f.strength for f in forces])
-    self.act_index = jp.array(act_index)
+    self.force_index = jp.array(force_index)
+    self.torque_index = jp.array(torque_index)
 
-  def apply_reduced(self, force: jp.ndarray) -> jp.ndarray:
+  def force_apply_reduced(self, force: jp.ndarray) -> jp.ndarray:
     dvel = force * self.strength / self.body.mass
     return dvel, jp.zeros_like(dvel)
```

Comment:
Suggestion for slightly simpler interface:  here, either `return dvel` and `return dang` for the torque fn bit, OR use a single function that takes both `force` and `torque` and return the dvel and dang bits together

Reply:
+1 - we have other parts of the code that only return dvel or dang, so not breaking any conventions by doing this.

Reply:
But actually I slightly prefer the second option: just have one function.  Also, don't forget to fix the return type!

```python
def apply_reduced(self, force: jp.ndarray, torque: jp.ndarray) -> Tuple[jp.ndarray, jp.ndarray]:
  ...
```

Reply:
@erikfrey do you still prefer the second option now that I added the mask field? It would make the convention quite clunky having all four inputs. For now I went with @cdfreeman-google 's suggestion of only returning what it needs. That also seems like a slightly cleaner abstraction since the calculation are entirely independent.